### PR TITLE
Add rubocop and setup rake ci

### PR DESCRIPTION
**Pull request by aaron-collier**
The large number of file changes here are from `rubocop --safe-auto-correct`.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
+# clear the default task injected by rspec
+task(:default).clear
+
+# and replace it with our own
+task default: [:ci]
```
Comment (jcoyne) [inline Rakefile]: just do `task default: [:rubocop, :rspec]`

Comment (aaron-collier) [reply Rakefile]: Ok, running directly in travis with this instead of a rake task.

Comment (jcoyne) [reply Rakefile]: 👍

**Comment by jcoyne** [comment]
Looks like this needs a rebase.

**Comment by aaron-collier** [comment]
@jcoyne rebase done :-)

**Pull request merged**